Tests/LibWeb/Text/input/include.js

```diff
@@ -17,13 +18,21 @@ document.addEventListener("DOMContentLoaded", function () {
 });
 
 function test(f) {
+    if(__alreadyCalledTest)
+        throw new Error("test() may only be called once per page");
+    __alreadyCalledTest = true;
+
     document.addEventListener("DOMContentLoaded", f);
     window.addEventListener("load", () => {
         internals.signalTextTestIsDone();
     });
 }
 
 function asyncTest(f) {
+    if(__alreadyCalledTest)
+        throw new Error("test() may only be called once per page");
+    __alreadyCalledTest = true;
```

Comment:
I think we also need to throw from `done`. otherwise single asyncTest could still call `signalTextTestIsDone` twice.